autoload/ollama/setup.vim

```diff
@@ -453,21 +455,25 @@ function! ollama#setup#Init() abort
         endif
         echon "\n"
 
-        if g:ollama_use_venv
+        if g:ollama_use_venv && g:ollama_embedded_python
```

Comment:
This check is wrong IMO.
embedded python is used for Edit functionality only (currently, more will come) where we use `python3 << EOF ...` code in VimScript.
The venv support is also needed for calling external vimscripts using `job_start` or `system`.
It may work for you when you have installed all dependencies on your system, but most users will use the plugin's built-in venv support, which makes setting up the plugin much easier.